composer.json

```diff
@@ -63,7 +64,7 @@
         "liip/imagine-bundle": "<1.9",
         "sonata-project/block-bundle": "<3.11 || >=4.0",
         "sonata-project/classification-bundle": "<3.0 || >= 5.0",
-        "sonata-project/formatter-bundle": "<3.2 || >= 4.0",
+        "sonata-project/formatter-bundle": "<3.4",
```

Comment:
sgould this be done in this PR?

Reply:
We only test ^3.4 deps for the formatter, so the other versions should be conflicted